### PR TITLE
Add basic video selection creation

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

File: media/js/src/AssetDetail.jsx [hunk 1]
```diff
@@ -136,35 +197,72 @@ export default class AssetDetail extends React.Component {
...
-                                                            <tr className="sherd-clipform-editing"><td><input type="text" className="timecode" id="clipStart" readOnly value="00:00:00" /><div className="helptext timecode">HH:MM:SS</div></td><td style={{width: '10px', textAlign: 'center'}}>-</td><td><input type="text" className="timecode" id="clipEnd" readOnly value="00:00:00" /><div className="helptext timecode">HH:MM:SS</div></td><td className="sherd-clipform-play"><input type="image" title="Play Clip" className="regButton videoplay" id="btnPlayClip" src="/media/img/icons/meth_video_play.png" /></td>
+                                                            <tr className="sherd-clipform-editing">
+                                                                <td>
+                                                                    <input
+                                                                        type="text" className="timecode" id="clipStart" readOnly
+                                                                        value={this.state.annotationStartTime} /><div className="helptext timecode">HH:MM:SS</div></td>
```
Comment (sdreher) [inline media/js/src/AssetDetail.jsx]: the annotationStartTime and annotationEndTime are coming through as floats...likely these should be cast to int.

Comment (nikolas) [reply media/js/src/AssetDetail.jsx]: Yeah, I definitely have some work to do there. I'll get that sorted out before a merge.

**Comment by nikolas** [comment]
Okay @sdreher this is in a messy, but mergeable state. A good stopping point before I add more work here. Feel free to review and add any comment or anything. 

* Create video selections
* Delete video selections

And these are updated dynamically in the view.

Still TODO (in future PRs):
* Actually viewing the selection video clips
* Add a "selection was created" bootstrap alert, on creation
* Fix timecode formatting - right now it's just a number / float. Needs to be HH:MM:SS text string on the front-end, and int on the backend.

File: media/js/src/AssetDetail.jsx [hunk 2]
```diff
@@ -14,18 +17,126 @@ import Projection from 'ol/proj/Projection';
...
+            'annotation-annotation_data': {
+                startCode: this.state.selectionStartTime,
+                endCode: this.state.selectionEndTime,
+                duration: 251,
+                timeScale: 1,
+                start: 0,
+                end: 99
+            }
+        }).then(function() {
+            console.log('selection created');
```
Comment (sdreher) [inline media/js/src/AssetDetail.jsx]: stray console.log

**Comment by sdreher** [comment]
I was able to verify add and delete functionality. :clap:

**Pull request merged**